src/user/mod.rs

```diff
@@ -1,28 +1,58 @@
+pub mod rust;
+
 use crate::parser::{self, NixNode};
 
 use std::{collections::HashMap, io::Cursor, str::FromStr};
 
 use parse_display::{Display, FromStr};
 use skim::prelude::*;
+use smol_str::SmolStr;
 
-//#[derive(Eq, PartialEq, Debug, Clone)]
-//pub struct UserResult {
-//user_selection: String,
-//}
+#[derive(Debug, Default, Clone, PartialEq, Eq, Display)]
+pub struct SmlStr(pub SmolStr);
```

Comment:
why do we introduce this alias instead of just re-exporting `SmolStr` using `pub use smol_str::SmolStr;`?